bot/exts/recruitment/talentpool/_review.py

```diff
@@ -78,14 +81,14 @@ def schedule_review(self, user_id: int) -> None:
 
     async def post_review(self, user_id: int, update_database: bool) -> None:
```

Comment:
The `-> None` here is kind of ... wrong. But since it was wrong before, no need to fix it here.

Reply:
How so?

Reply:
The annotation seems correct here